packages/sitecore-jss-angular/src/lib.module.ts

```diff
@@ -81,14 +82,19 @@ export class JssModule {
   }
 
   /** Instantiates a module for a lazy-loaded JSS component */
-  static forChild(component: Type<any>): ModuleWithProviders {
+  static forChild(components: Array<Type<any>>): ModuleWithProviders {
```

Comment:
This would appear to be a breaking signature change (and would thus also break the existing angular sample app, which does not pass an array). Can this be implemented such that the array is optional?

Reply:
Yes, I done this in next commit.